src/js/components/addData/AddDataHeader.jsx

```diff
@@ -8,15 +8,15 @@ import moment from 'moment';
 
 import * as ReviewHelper from '../../helpers/reviewHelper.js';
 
-class LastUpdated extends React.Component {
-    constructor(props){
-        super(props);
-    }
-    
+class SubmissionContext extends React.Component {
     render() {
         return (
             <div className="last-updated">
-                Last Saved: {this.props.last_updated}
+                Last Saved: {this.props.formattedTime}
+                <br />
+                {this.props.agencyName}
+                <br />
+                {this.props.submissionTime}
```

Comment:
`submissionTime` will be `Q3/2016` for quarterly submissions as a date for month submissions, so I'm not sure of a great variable name. `timePeriodLabel`?